cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,50 @@
+import PageObject from '../PageObject';
+
+class settingsPageObject extends PageObject {
+  url = '/settings';
+
+  clickOnUpdate() {
+    cy.getByDataCy('settings-btn-update')
+      .click();
+  }
+
+  get usernameField() {
+    return cy.getByDataCy('settings-username');
+  }
+
+  typeUsername(name) {
+    this.usernameField
+      .clear()
+      .type(name);
+  }
+
+  typeBio(bio) {
+    cy.getByDataCy('settings-bio')
+      .type(bio);
+  }
+
+
+  typeEmail(email) {
+    cy.getByDataCy('settings-Email')
```

Comment:
Data-cy attribute name should be in lowercase. Replace 'settings-Email' with 'settings-email'.

Reply:
done